site/_docs/history.md

```diff
@@ -43,6 +43,96 @@ z.
 #### Breaking Changes
 {: #breaking-1-37-0}
 
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 19;
+Guava versions 21.0 to 32.1.3-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-37-0}
```

Comment:
This should be `1-38-0` (same for the others).

Reply:
done